### DISCUSSION TITLE
Add Android App to F-Droid

### DISCUSSION THREAD

**Discussion by shortwavesurfer2009** [BearWare/TeamTalk5#1769] [Ideas]
### Description

If the app is open source it would be great to see it in F-Droid for those of us who don't wish to corrupt our devices eith the vile google play store.

### Application

- [ ] qtTeamTalk
- [x] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by bear101** [comment]
This is not something we plan. Google Play is an excellent platform for distributing software and getting feedback.
